theme/lumo/vaadin-tab-styles.html

```diff
@@ -189,6 +189,40 @@
         box-shadow: inset 0 0 0 2px var(--lumo-primary-color-50pct);
         border-radius: var(--lumo-border-radius);
       }
+
+      /* RTL specific styles */
+
+      :host([dir="rtl"])::before,
+      :host([dir="rtl"])::after {
+        left: auto;
+        right: 50%;
+        transform: translateX(50%) scale(0);
+      }
+
+      :host([dir="rtl"][selected]:not([orientation="vertical"]))::before,
+      :host([dir="rtl"][selected]:not([orientation="vertical"]))::after {
+        transform: translateX(50%) scale(1);
+      }
+
+      :host ::slotted(iron-icon:first-child) {
+        margin-right: 0;
+      }
+
+      :host ::slotted(iron-icon:last-child) {
+        margin-left: 0;
+      }
```

Comment:
Should these be scoped to `[dir="rtl"]`?

Reply:
Good catch! Updated selectors